plugin/pkg/scheduler/algorithm/predicates/predicates_test.go

```diff
@@ -2612,11 +2620,15 @@ func TestInterPodAffinityWithMultipleNodes(t *testing.T) {
 	labelRgUS := map[string]string{
 		"region": "US",
 	}
+	type NodeTest struct {
+		node                         v1.Node
+		expectAffinityFailureReasons []algorithm.PredicateFailureReason
+	}
 
 	tests := []struct {
 		pod    *v1.Pod
 		pods   []*v1.Pod
-		nodes  []v1.Node
+		nodes  []NodeTest
```

Comment:
optional: instead of a new type ("NodeTest"), you could have added another slice that would list the expected failure reasons in the same order as the nodes array.

Reply:
reasonable,done